js/validate.js

```diff
@@ -27,6 +29,7 @@ async function handleSubmit(event) {
       if (response.ok) {
         status.innerHTML = 'Thanks for your submission!';
         form.reset();
+        LOCAL_STORAGE_KEY.setItem(LOCAL_STORAGE_KEY, null);
```

Comment:
I am very sure this will not work.

Can you test this? Submit a form successfully and check your console.

Reply:
![image](https://github.com/moshoodabdullahi/portfolio/assets/123197179/258a8a42-8a76-47fd-b23d-863b2a4af75c)

Reply:
Done